kuzzleio/kuzzle
# [fix] Default Vault secrets file location

## What does this PR do ?
A minor breaking change has been introduced in the 2.2.0 Kuzzle release. The file tree organization move the default Vault secrets file location from `config/secrets.enc.json` to `lib/config/secrets.enc.json`. This PR fixes this file location. 

### How should this be manually tested?
Create a Vault secrets file and put it in `config/` with the name `secrets.enc.json`. Start Kuzzle providing the vault key. It should works correctly without forcing the Vault secrets file location using `KUZZLE_SECRETS_FILE` for example.
